addons/sys_rack/fnc_rackPFH.sqf

```diff
@@ -0,0 +1,73 @@
+/*
+ * Author: ACRE2Team
+ * Per frame execution. Sets if player is inside a vehicle and manages the access to rack radios.
+ *
+ * Arguments:
+ * 0: Array of arguments <ARRAY>
+ *  0: Player unit <OBJECT>
+ *  1: Vehicle with intercom <OBJECT>
+ * 1: PFH unique identifier <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [[player, vehicle player], 12] call acre_sys_intercom_fnc_intercomPFH
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_param", "_handle"];
+
+_param params ["_player", "_vehicle"];
+
+// Check if the player entered a position with a rack already active in intercom
+if (_player != vehicle _player) then {
+    {
+        private _radioId = [_x] call FUNC(getMountedRadio);
+        if (_radioId != "" && {!(_radioId in ACRE_ACCESSIBLE_RACK_RADIOS || _radioId in ACRE_HEARABLE_RACK_RADIOS)}) then {
+            private _functionality = [_radioId, _vehicle, _player, _x] call EFUNC(sys_intercom,getRxTxCapabilities);
+            if (_functionality != RACK_NO_MONITOR) then {
+                // Add the radio to the active list since it is already active in the intercom system
+                [_vehicle, _player, _radioId] call FUNC(startUsingMountedRadio);
+            };
+        };
+    } forEach (([_vehicle, _player] call FUNC(getHearableVehicleRacks)) apply {toLower _x});
```

Comment:
Just do `toLower` in the actual loop, this is effectively looping through hearable vehicle racks array twice as it is now.
```sqf
{
    private _hearableVehicleRack = toLower _x;
    // rest of code
} forEach ([_vehicle, _player] call FUNC(getHearableVehicleRacks));
```

Reply:
This is removed in #434. So this is not going to be in here.